CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# How to Contribute
+
+Thanks for your interest in Noda Time. We appreciate all kinds of contributions, from submitting issues to improving documentation; from writing tests to implementing new code. All help is welcome!
+
+## Basic Requirements
+
+If you want to contribute to the codebase, you must have Visual
```

Comment:
I think it would be more accurate to *recommend* using Visual Studio. You *need* the .NET Core SDK, but that's all - it's feasible to work on Noda Time just from the command line, or from VS Code.

Reply:
Poorly worded. Will fix, thanks.